app/pages/gsoc/gsoc2022.js

```diff
@@ -0,0 +1,91 @@
+import Head from "next/head";
```

Comment:
Make sure this file's name is the same as the CSS file --- Leaderboard 

You can parameterize the Leaderboad component with the gsoc2022 URL.